dll/win32/wtsapi32/wtsapi32.c

```diff
@@ -302,23 +321,64 @@ BOOL WINAPI WTSQuerySessionInformationW(
     LPWSTR* Buffer,
     DWORD* BytesReturned)
 {
+    switch (WTSInfoClass)
+    {
+        case WTSUserName:
+        {
+            WCHAR* username;
+            DWORD count = 0;
+
+            GetUserNameW(NULL, &count);
+            if (GetLastError() != ERROR_INSUFFICIENT_BUFFER)
+                return FALSE;
+            if (!(username = heap_alloc(count * sizeof(WCHAR))))
+                return FALSE;
+            GetUserNameW(username, &count);
+            *Buffer = username;
+            *BytesReturned = count * sizeof(WCHAR);
+            return TRUE;
+        }
+
+        case WTSConnectState:
+        {
+            WCHAR* output;
+
+            // Windows always returns 4 bytes even on x64 platform
+            const DWORD size = 4;
```

Comment:
`sizeof(DWORD)` seems clearer, and would seem to eliminate the need for the comment.